documentation/install_from_packages.markdown

```diff
@@ -61,18 +60,8 @@ Use Puppet to install PuppetDB:
 Step 4: Configure Database
 -----
 
-- Most deployments should
-  [set up a PostgreSQL server and configure PuppetDB to use it][configure_postgres]. You
-  may also need to [adjust the maximum heap size][configure_heap].
-
-- For the moment, PuppetDB also supports an embedded HSQLDB database,
-  but this support has been deprecated and will be removed in a future
-  release. Please see [this email][hsqldb_deprecation_mail] to the
-  puppet-users list for further information.  If you do choose HSQLDB,
-  you should [increase PuppetDB's maximum heap size][configure_heap]
-  to at least 1 GB.
-
-You can change PuppetDB's database at any time, but note that changing the database does not migrate PuppetDB's data, so the new database will be empty. However, as this data is automatically generated many times a day, PuppetDB should recover in a relatively short period of time.
+- [Set up a PostgreSQL server and configure PuppetDB to use it][configure_postgres].
+  You may also need to [adjust the maximum heap size][configure_heap].
```

Comment:
Does this configure_heap part need to be there? Seems like it's not related to configuring the database anymore